packit_service/worker/handlers/open_scan_hub.py

```diff
@@ -99,17 +100,69 @@ class OpenScanHubTaskFinishedHandler(
     event: OpenScanHubTaskFinishedEvent
     task_name = TaskName.openscanhub_task_finished
 
+    @staticmethod
+    def check_new_findings_found(url: str) -> bool:
```

Comment:
May be use the term "identified" rather than "found" to avoid confusion with words. Or just the name `check_new_findings`.